test/components/DateInput_spec.jsx

```diff
@@ -89,6 +89,28 @@ describe('DateInput', () => {
     });
   });
 
+  describe('#componentWillReceiveProps', () => {
+    describe('nextProps.displayValue exists', () => {
+      it('sets state.displayString to \'\'', () => {
+        const dateString = 'foo123';
+        const wrapper = shallow(<DateInput id="date" />).dive();
+        wrapper.setState({ dateString });
+        wrapper.instance().componentWillReceiveProps({ displayValue: '1991-07-13' });
+        expect(wrapper.state()).to.have.property('dateString', '');
+      });
+    });
+
+    describe('nextProps.displayValue does not exist', () => {
+      it('does not change state.displayString', () => {
```

Comment:
Same here with `dateString` instead of `displayString`.

Reply:
You are right! Will fix